libcontainer/SPEC.md

```diff
@@ -367,7 +367,7 @@ profile <profile_name> flags=(attach_disconnected,mediate_deleted) {
 }
 ```
 
-*TODO: seccomp work is being done to find a good default config*
+*Seccomp filtering is supported with configurable default profiles*
```

Comment:
Not sure what you mean with a configurable default profile. There is no default profile AFAIK.

Also, considering how extensive this doc tends to be, maybe we can add some example seccomp profile here?

Reply:
I was about to comment the same thing -- this is talking about the default configuration for runc, and we don't have a default seccomp profile. Upstream runtimes have their own profiles but we do not yet have a default one.

But yes, a better PR would be to provide an example (maybe just taking the Podman one? I wouldn't use the Docker one since it still suffers from the `EPERM` issue described in https://github.com/moby/moby/issues/42871.).